sdk/helper/certutil/types.go

```diff
@@ -689,6 +689,20 @@ const (
 	PermitNotAfterBehavior
 )
 
+func (n NotAfterBehavior) String() string {
+	names := map[NotAfterBehavior]string {
```

Comment:
nit: Should this be pulled out into a "const" var (I know an actual const isn't possible).

Reply:
We could refactor it when we need it, since this is in the SDK I'd rather not expose it until we need to.

Reply:
Ah, I see on godbolt that could impact the performance, so I'll move it to a private thingy.